django_ipam/base/fields.py

```diff
@@ -27,7 +27,7 @@ def to_python(self, value):
 
         try:
             network = ip_network(value, strict=False)
-        except ValueError as e:
+        except ValueError:
```

Comment:
Also, why was this changed in this commit, doesn't look like it's related to the issue, any particular reason? :smile:

Reply:
maybe because of a `flake8` warning since that `e` doesn't seem to be used